### DISCUSSION TITLE
Alist无法挂载阿里云盘问题分析

### DISCUSSION THREAD

**Discussion by dycfan** [AlistGo/alist#3391] [General]
从2月13日开始，Alist挂载的阿里云盘开始无法访问存储中的文件下载直链地址（failed get link: invalid X-Device-Id）。
按照官方文档的指引，用两种方法分别获取到refreshToken重新配置，依然无法访问到存储中的文件下载直链地址（failed get link: invalid X-Device-Id）。
不仅仅是Alist，CloudDrive挂载阿里云盘成功后，依然无法访问阿里云盘存储中的文件下载直链地址（服务器关闭）。
截至目前，阿里云盘的第三方客户端工具仅仅当贝播放器能正常连接到阿里云盘并正常获取到存储中的文件下载（播放）直链地址。
从以上情况看，要修复Alist对阿里云盘的支持问题，需要项目方寻求到阿里云盘官方支持并付出更多努力。
供参考。

**Comment by dreamsamethyst** [comment]
这个支持感觉不太可能了，不然阿里云盘的VIP就没有任何用处了

**Comment by u-lee-xu** [comment]
看来是......
